frontend/components/text-classifier/sidebar/TextClassificationViewInfo.vue

```diff
@@ -62,12 +55,20 @@
 <script>
 import "assets/icons/support";
 export default {
-  data: () => ({
-    showHelpPanel: false,
-  }),
+  props: {
+    dataset: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    visibleViewInfo() {
+      return this.dataset?.viewSettings.visibleViewInfo;
+    },
```

Comment:
in case there will be a problem to reach visibleViewInfo, it will be nice to have a default value in case it's undefined => you can replace by return this.dataset?.viewSettings.visibleViewInfo || false;